arrow-integrations-retrofit-adapter/src/main/kotlin/arrow/integrations/retrofit/adapter/io/TaglessAdapterFactory.kt

```diff
@@ -0,0 +1,38 @@
+package arrow.integrations.retrofit.adapter.io
+
+import arrow.Kind
+import arrow.fx.IO
+import arrow.fx.IOPartialOf
+import arrow.fx.extensions.io.async.async
+import arrow.fx.typeclasses.Async
+import arrow.integrations.retrofit.adapter.parseTypeName
+import retrofit2.CallAdapter
+import retrofit2.Retrofit
+import java.lang.reflect.ParameterizedType
+import java.lang.reflect.Type
+
+class TaglessAdapterFactory<F>(private val async: Async<F>) : CallAdapter.Factory() {
+
+  companion object {
+    fun createIO(): TaglessAdapterFactory<IOPartialOf<Throwable>> = TaglessAdapterFactory(IO.async())
```

Comment:
Throwable is implicit in IO so I think you never have to use IO<Throwable, Response> but instead IO<Nothing, Response>

Reply:
Oh, yes
Now I see that `ApplicativeError.raiseError` use `IO.raiseException`, not `IO.raiseError` like I thought
Changed it and written tests for failed flow of usage
Now, having one bug, I will fix it and push all of this :D